src/wwwroot/js/genpage/browsers.js

```diff
@@ -549,7 +549,7 @@ class GenPageBrowserClass {
             if (!this.showDisplayFormat) {
                 formatSelector.style.display = 'none';
             }
-            let buttons = createSpan(`${this.id}-button-container`, 'browser-header-buttons', 
```

Comment:
what

Reply:
empty space at the end of the line. my editor autoformatted that. I had earlier code implementation here that I removed, and must not have noticed all the empty line removal

Reply:
In vscode, trailing whitespaces are by default removed. To prevent that behavior this setting is needed: `files.trimTrailingWhitespace: false`
I went ahead and reverted removing those whitespaces accidentally.